releasenotes.md

```diff
@@ -1,6 +1,17 @@
 GeoIP2 .NET API Release Notes
 =============================
 
+2.4.0-beta1 (2015-09-10)
+------------------------
+
+* Async support was added to the `WebServiceClient`. Each web-service end
+  point now has a corresponding *Async(ip)` method. GitHub #1.
+* Use of RestSharp was replaced by `HttpWebRequest` for synchronous HTTP
+  requests and `HttpClient` for asynchronous requests. Microsoft BCL libraries
+  are used to provide `async`/`await` and `HttpClient` support on .NET 4.0.
+  GitHub #32.
```

Comment:
Woops. That is a typo. It should be #33 that was fixed. The second sentence is explaining new dependencies, MS's BCL, caused by the changes in the first. This library supports .NET 4.0, which doesn't include `await`, `async`, or `HttpClient` directly.